apps/utils/BaseTableView.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.contrib.postgres.search import TrigramSimilarity
+from django.db.models import Q
+from django_tables2 import SingleTableView
+
+
+class BaseTableView(SingleTableView, PermissionRequiredMixin):
+    paginate_by = 25
+    template_name = "table/single_table.html"
+
+    def get_queryset(self):
```

Comment:
This filtering here is quite specific to the `ExperimentModel` so I think this class should be renamed `BaseExperimentTableView`